go/vt/vtadmin/http/request.go

```diff
@@ -77,6 +77,25 @@ func (r Request) ParseQueryParamAsUint32(name string, defaultVal uint32) (uint32
 	return defaultVal, nil
 }
 
+// ParseQueryParamAsInt32 attempts to parse the query parameter of the given
+// name into a uint32 value. If the parameter is not set, the provided default
+// value is returned.
+func (r Request) ParseQueryParamAsInt32(name string, defaultVal int32) (int32, error) {
+	if param := r.URL.Query().Get(name); param != "" {
```

Comment:
Styling-wise, I'd prefer to first handle the case where `param == ""` and do an early return, then unindent the rest of the function. But it's insignificant and not worth the rebuild time.

Reply:
@shlomi-noach Copy-pasted this mostly from another function for `uint32`, so it's not "new" in that sense.